dracut/modules.d/90kiwi-dump/kiwi-dump-image.sh

```diff
@@ -10,7 +10,7 @@ type fetch_file >/dev/null 2>&1 || . /lib/kiwi-net-lib.sh
 #--------------------------------------
 function report_and_quit {
     local text_message="$1"
-    run_dialog --timeout 60 --msgbox "\"${text_message}\"" 5 70
+    run_dialog --timeout 60 --msgbox "\"${text_message}\"" 5 100
```

Comment:
We probably can't widen this more than 80 columns due to potential limitations with serial consoles...

Reply:
I've tested on serial consoles and interesting enough it did not harm if the width is set wider than the actual console. I guess there is code in dialog that checks that. But 80 is fine by me too.